src/index.js

```diff
@@ -221,16 +221,18 @@ function debounce(func, wait, immediate) {
   }
 }
 
+const debouncedLoadSearchResults = debounce(() => {
+  const searchBar = document.getElementById('search')
+  loadSearchResults(searchBar)
+}, 450)
```

Comment:
While we're at it, could you please extract this into a top-level `const` and add a comment on how you got this value (i.e., link to your blog post!)

Reply:
sure!